From bf55ae4b94f0c996b8df3506801d9a8da3f1950a Mon Sep 17 00:00:00 2001
From: Ioan-Adrian Ratiu <adi@adirat.com>
Date: Thu, 12 Apr 2018 11:47:15 +0300
Subject: [PATCH] config.mk: make safe for cross-compilation within OE

Signed-off-by: Ioan-Adrian Ratiu <adi@adirat.com>
---
 config.mk | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/config.mk b/config.mk
index 9fb18da..f3907ea 100644
--- a/config.mk
+++ b/config.mk
@@ -2,15 +2,7 @@
 VERSION = 0.0
 
 # paths
-PREFIX = /usr/local
+PREFIX = /usr/bin
 MANPREFIX = $(PREFIX)/share/man
 
-CC = cc
-AR = ar
 RANLIB = ranlib
-
-# for NetBSD add -D_NETBSD_SOURCE
-# -lrt might be needed on some systems
-CPPFLAGS = -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_XOPEN_SOURCE=700 -D_FILE_OFFSET_BITS=64
-CFLAGS   = -std=c99 -Wall -pedantic
-LDFLAGS  = -s
-- 
2.16.3

